MVPSt_2023_Jan/MVPSt_2023_Jan/Pages/HomePage.cs

```diff
@@ -1,177 +1,276 @@
+            educationuniversityTextbox.SendKeys(University);
 
             // select country from Country drop-down list
             // create object for new SelectElement class
-            IWebElement countryDropDownElement = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[4]/div/div[2]/div/div/div[1]/div[2]/select"));
-            SelectElement selectacountry = new SelectElement(countryDropDownElement);
+            //IWebElement countryDropDownElement = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[4]/div/div[2]/div/div/div[1]/div[2]/select"));
+            //SelectElement selectacountry = new SelectElement(countryDropDownElement);
+            SelectElement selectacountry = new SelectElement(educationcountryDropDown);
             selectacountry.SelectByText(Country);
 
             // select title from Title drop-down list
             // create object for new SelectElement class
-            IWebElement titleDropDownElement = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[4]/div/div[2]/div/div/div[2]/div[1]/select"));
-            SelectElement selectatitle = new SelectElement(titleDropDownElement);
+            //IWebElement titleDropDownElement = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[4]/div/div[2]/div/div/div[2]/div[1]/select"));
+            //SelectElement selectatitle = new SelectElement(titleDropDownElement);
+            SelectElement selectatitle = new SelectElement(educationtitleDropDown);
             selectatitle.SelectByText(Title);
 
             // find and enter the Degree
-            IWebElement eddegreeTextbox = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[4]/div/div[2]/div/div/div[2]/div[2]/input"));
-            eddegreeTextbox.SendKeys(Degree);
+            //IWebElement eddegreeTextbox = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[4]/div/div[2]/div/div/div[2]/div[2]/input"));
+            educationdegreeTextbox.SendKeys(Degree);
 
             // select year of graduation from drop-down list
             // create object for new SelectElement class
-            IWebElement yearofgradDropDownElement = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[4]/div/div[2]/div/div/div[2]/div[3]/select"));
-            SelectElement selectayearofgrad = new SelectElement(yearofgradDropDownElement);
+            //IWebElement yearofgradDropDownElement = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[4]/div/div[2]/div/div/div[2]/div[3]/select"));
```

Comment:
remember to remove comments :)

Reply:
I removed many, many comments from all files but probably overlooked a few